ui/packages/consul-ui/tests/acceptance/dc/services/show/topology/stats.feature

```diff
@@ -33,4 +33,50 @@ Feature: dc / services / show / topology / stats
     ---
     And I see topology on the tabs
     And I see the "[data-test-topology-metrics-stats]" element
+  Scenario: Given metrics is enabled, all metrics are disabled for an ingress gateway Topology
+    Given 1 datacenter model with the value "dc1"
+    Given a "prometheus" metrics provider
+    And 1 node models
+    And 1 service model from yaml
+    ---
+    - Service:
+        Name: ingress-gateway
+        Kind: ingress-gateway
+        ID: ingress-gateway-with-id
+    ---
+    When I visit the service page for yaml
+    ---
+      dc: dc1
+      service: ingress-gateway
+    ---
+    And I see topology on the tabs
+    And I don't see the "[data-test-topology-metrics-stats]" element
+    And I see the "[data-test-sparkline]" element
+  Scenario: Given metrics is enabled, metric stats are disabled for ingress gateway as downstream services
+    Given 1 datacenter model with the value "dc1"
+    Given a "prometheus" metrics provider
+    And 1 node models
+    And 1 service model from yaml
+    ---
+    - Service:
+        Name: service-0
+        ID: service-0-with-id
+    ---
+    And 1 topology model from yaml
+    ---
+      Upstreams: []
+      Downstreams:
+        - Name: db
+          Namespace: default
```

Comment:
```suggestion
          Namespace: @namespace
```
That ^ should make it replace `@namespace` with whatever namespace is currently being tested (enterprise tests will run this test for an empty namespace, the `default` namespace, a `team-1` namespace and an `undefined` namespace.

Reply:
I've updated this.